### ISSUE TITLE
Hide menu features if respective plugin is not installed

### ISSUE THREAD

**Issue by alanmels** [virtualmin/virtualmin-gpl#235]
You won't see _Switch to Server's Admin_ option unless Webmin login for the respective user is enabled. Which is right. However, there are number of other menu items which are always there regardless if respective features are installed and enabled or not. 

For example, _FTP directory restrictions _ under _Limits and Validation_ are always there even if ProFTP server is not installed at all. The same goes for _Cloudmin Services Client_ - we never used it, but it's always hanging there - I believe that kind of menu options could be additionally enabled for those who intend to use them, but not for absolutely majority of Virtualmin users who never use them.

**Comment by jcameron** [comment]
Good point about FTP directory restrictions - we'll remove that.

The "Cloudmin Services Client" option can't really be hidden, because we can't tell if Cloudmin is installed on another machine or not.

**Comment by alanmels** [comment]
> Good point about FTP directory restrictions - we'll remove that.

Thanks for taking the suggestion positively.

> The "Cloudmin Services Client" option can't really be hidden, because we can't tell if Cloudmin is installed on another machine or not.

Not to argue, but if the menu item was hidden by default, then those who want to use Cloudmin would just enable the hidden menu item.

**Comment by jcameron** [comment]
We don't really have a facility yet to hide menu items, unless there is a separate configuration somewhere that controls if they are shown.  One could be added in this case, but it might make the feature harder to find for users who want it.

**Comment by alanmels** [comment]
Fair enough, removing _FTP directory restrictions_ is already progress. Thanks!

**Comment by jcameron** [comment]
Anything else you noticed that should be hidden but isn't currently?

**Comment by iliaross** [comment]
❗️ If the issue isn't resolved, feel free to reopen it with updated details.
